### ISSUE TITLE
had to change the terraform.yml file to make it work

### ISSUE THREAD

**Issue by munnep** [hashicorp/learn-terraform-github-actions#1031]
I had to change the terraform.yml file to make it work. 

Tried creating a pull request but it immediately got closed.

Changed the following

from this:
```
jobs:
  terraform:
    name: "Terraform"
    runs-on: ubuntu-latest
    permissions:
      pull-request: write
```
to this
```
jobs:
  terraform:
    name: "Terraform"
    runs-on: ubuntu-latest
    permissions: write-all
```

**Comment by lucascantor** [comment]
I proposed a fix for this a few days ago, but it was auto-closed: https://github.com/hashicorp/learn-terraform-github-actions/pull/1021

**Comment by alanszlosek** [comment]
Thank you again! I've merged #1021
